mkosi

```diff
@@ -3113,7 +3113,10 @@ def run_build_script(args, workspace, raw):
             cmdline.append("--setenv=BUILDDIR=/root/build")
             cmdline.append("--bind=" + args.build_dir + ":/root/build")
 
-        if not args.with_network:
+        if args.with_network:
+            # If we're using the host network namespace, use the same resolver
+            cmdline.append("--bind-ro=/etc/resolv.conf")
```

Comment:
Side note: I changed this from `cmdline += […]` to `cmdline.append(…)` to match the surrounding code in `run_build_script`, but I suppose at some point we should go over the whole program to reduce inconsistencies like this (e. g. the multiple ways to format strings we use), and perhaps look into some CI that makes sure we stick to one style.